coordinator/cql/src/main/java/org/apache/cassandra/stargate/transport/internal/CQLMessageHandler.java

```diff
@@ -0,0 +1,529 @@
+/*
```

Comment:
Would it make sense to add some marker for files that are direct copies as base -- specific comment pointing to original file path (I know, it's only different by ".internal" part)?
And then changes within -- except for necessary `import` diffs -- marked separately?

Or is it too much noise if and when all classes under `internal` are copies?
I just try to think of what would make it easier for team members who are less familiar (like ones joining now) with existing codebases to figure out relationship between files and repos we have.